### PR TITLE
Splitted two booking flows to separate services

### PR DESCRIPTION
**Pull request by mingazhev**
Credit card and account flows differ much and do not have real duplicate code so I've splitted them to separate small services.
+ in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time.
This is discussable

### CODE REVIEW

File: Api/Services/Accommodations/Bookings/Flows/ICreditCardBookingFlow.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using HappyTravel.Edo.Api.Models.Agents;
+using HappyTravel.Edo.Api.Models.Bookings;
+
+namespace HappyTravel.Edo.Api.Services.Accommodations.Bookings.Flows
+{
+    public interface ICreditCardBookingFlow
```
Comment (kirillta) [inline Api/Services/Accommodations/Bookings/Flows/ICreditCardBookingFlow.cs:8]: Both class names are ambiguous. What about `IBankCardBookingFlow` and `IFinancialAccountBookingFlow`?

**Comment by kirillta** [comment]
> * in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time

That's OK while one cannot cancel such bookings

File: Api/Services/Accommodations/Bookings/Flows/ICreditCardBookingFlow.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/Flows/ICreditCardBookingFlow.cs:8]: Renamed

**Comment by mingazhev** [comment]
> 
> 
> > * in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time
> 
> That's OK while one cannot cancel such bookings

Good note, reverted money capture, at least for now

**Review by kirillta — APPROVED** [review]

**Pull request merged**